src/ucs/sys/event_set.h

```diff
@@ -9,27 +9,34 @@
 
 #include <ucs/type/status.h>
 
+#include <stdint.h>
+
+
 /**
  * ucs_sys_event_set_t structure used in ucs_event_set_XXX functions.
  *
  */
 typedef struct ucs_sys_event_set ucs_sys_event_set_t;
 
+/**
+ * Bit set composed using the available event types
+ */
+typedef uint8_t ucs_event_set_types_t;
```

Comment:
why not use ucs_sys_event_set_t everywhere, e.g func params ? (maybe except tcp_ep where we can use uint8_t for reducing its size)

Reply:
`ucs_sys_event_set_t` is a structure that wraps epoll fd
I've defined `ucs_event_set_types_t` to use it instead of `ucs_event_set_type_t` and `int`

Reply:
right, i've meant `ucs_event_set_type_t`

Reply:
`ucs_event_set_type_t` defines the EVREAD/EVWRITE/etc, but to use the bitmask of event types (e.g. EVREAD|EVWRITE), we have to use uint8_t/int/etc. So, I've defined `ucs_event_set_types_t` to unify it and when it will be changed to `uint16_t` - no need to fix all occurrences.

Reply:
ok